app/views/assets/samples_partials/_tube_rack_samples.erb

```diff
@@ -0,0 +1,48 @@
+<section id="samples-table" class='info-panel'>
+  <header>
+    <a href='#plate-samples-table-collapser' role="button" data-toggle='collapse'>
```

Comment:
multiple mentions of 'plate' in the ids and hrefs in this file

Reply:
Yes, changing it